core/tests/looking_glass/test_cheshire_cat.py

```diff
@@ -0,0 +1,13 @@
+from cat.looking_glass.cheshire_cat import CheshireCat
+
+def test_get_base_url():
+    cat = CheshireCat()
+    assert cat.get_base_url() == "localhost"
+
+def test_base_path():
+    cat = CheshireCat()
+    assert cat.get_base_path() == "/app"
+
+def test_plugin_path():
+    cat = CheshireCat()
+    assert cat.get_plugin_path() == "/app/plugins"
```

Comment:
should be `/app/cat/plugins`

Reply:
static_path should be `/app/cat/static` and static_url `http://localhost:1865/static/` 

Very useful for plugins to store and give URLs of static files

Reply:
Added even the static path and fixed the path resolution